.travis.yml

```diff
@@ -11,6 +11,7 @@ after_success:
       if [ -n "$TRAVIS_PULL_REQUEST_SHA" ]; then
         npm run travis-deploy-once "npm run deploy-preview -- -d braid-design-system--${TRAVIS_PULL_REQUEST_SHA}.surge.sh";
         npm run travis-deploy-once "npm run post-commit-status";
+        npm run semantic-release -- --dry-run;
```

Comment:
If not a PR, so a branch push, and its not master, try and semantic release?

Reply:
This is just a temporary test for the new semantic-release config.

Reply:
Thanks guys! 🎉